src/wesconfig.h

```diff
@@ -40,11 +40,12 @@
   #undef VERSION
 #endif
 
-#define VERSION "1.15.14+dev"
+// TODO: undo changes to this file before merge
```

Comment:
Don't forget!

Reply:
I find it easier to remember if I put changes like this in a separate commit. That would also make the other changes easier to cherry-pick to a master-based branch for testing.

Reply:
/me prods @Pentarctagon with this "put changes like this in a separate commit" idea